### DISCUSSION TITLE
Any idea of how to change the trackpad scrolling speed in a game with no settings for it?

### DISCUSSION THREAD

**Discussion by josepheiba** [Whisky-App/Whisky#605] [Q&A]
I play Master Duel and when I scroll using the trackpad, it scrolls too fast I hit the bottom, I always have to scroll using the scrollbar. Any way I can change the scroll speed settings for the steam game/bottle? thanks

**Comment by rockyslaymaker** [comment]
I use https://linearmouse.app/ to customize mouse and scroll speed. Sadly it doesn't recognize apps launched through Whisky so you have to just modify global settings.
